core-bundle/src/EventListener/CommandSchedulerListener.php

```diff
@@ -82,7 +84,8 @@ private function canRunController(Request $request): bool
     private function canRunDbQuery(): bool
     {
         try {
-            return $this->connection->isConnected() && $this->connection->getSchemaManager()->tablesExist(['tl_cron']);
+            return $this->connection->isConnected()
+                && $this->connection->getSchemaManager()->tablesExist(['tl_cron_job']);
```

Comment:
This sounds like something the `Cron` should check? Though it also sounds like a performance penalty…

Reply:
No, this is strictly something the `CommandSchedulerListener` should check. The cron should only be executed, if Contao has been installed completely. I can dig up the issue why this was added if you want.